client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -92,11 +92,21 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                     </div>
                 </div>
                 <!-- editor -->
-                <div class="panel--editor" ng-show="panelState === 'editor'">
-                    <header>
-                        Editor<button class="btn btn-info pull-right" ng-click="togglePreview()" translate>{{ preview? 'Edit': 'Preview' }}</button>
-                        <button class="btn pull-right" ng-click="askAndResetEditor()" translate>Reset</button>
-                    </header>
+                    <div class="panel--editor" ng-show="panelState === 'editor'">
+                        <div ng-if="!preview">
+                        <header>
+                            Editor<button class="btn btn-info pull-right" ng-click="togglePreview()" ng-disabled="actionStatus() && !isCurrentPostPublished()" translate>Preview
```

Comment:
isCurrentPostPublished() does what it says. The && !isCurrentPostPublished() also requires the post to NOT be published in order for the preview button to be disabled. That means that if the post is already published, the preview button will always be enabled